scrapers/bank_of_zambia.py

```diff
@@ -0,0 +1,39 @@
+import requests
+from playwright.sync_api import sync_playwright
+from utils import download_filepath, get_browser, get_md5hash, parse_arguments
+
+BASE_URL = "https://www.boz.zm/"
+CIRCULAR_URL = f"{BASE_URL}circulars.htm"
+
+
+def bank_of_zambia_scraper():
+    with sync_playwright() as pw:
+        browser = get_browser(pw)
+        page = browser.new_page()
+        page.goto(CIRCULAR_URL)
+        session = requests.Session()
+
+        pdf_link = page.locator("div.list-group > a").all()[0]
+        link = pdf_link.get_attribute("href")
+        link = ""
+
+        if not link or not link.endswith(".pdf"):
+            browser.close()
+            raise ValueError("Invalid link or not found")
```

Comment:
Since you will need to make some changes anyway, let's clean up the error message here and in the other scraper as well to make it clearer.

```suggestion
        if not link or not link.endswith(".pdf"):
            browser.close()
            raise ValueError("Invalid file URL.")
```